app/lib/services.rb

```diff
@@ -4,6 +4,10 @@
 require "gds_api/email_alert_api"
 
 module Services
+  def self.frontend
+    Plek.find("frontend")
+  end
```

Comment:
I think just using `Plek.find("frontend")` in the concern would be clearer.

Reply:
That seemed inconsistent with what we've done before.
BUT! looking again I see that's because with the other services we're either doing initalisation calls or using require to bring other stuff in.

This does neither so makes sense, will amend